pkg/project/options.go

```diff
@@ -60,8 +63,57 @@ func FromConfig(c *Config) (*Project, error) {
 		}
 	}
 
+	for _, c := range c.Containers {
+		if c.Dockerfile != "" {
```

Comment:
Should we consider config to be invalid in this case and supply a specific error (or collection of errors).